tests/trx_generator/main.cpp

```diff
@@ -124,8 +124,8 @@ int main(int argc, char** argv) {
 
       if(vmap.count("accounts")) {
          boost::split(account_str_vector, accts, boost::is_any_of(","));
-         if(!transaction_specified && account_str_vector.size() < 2) {
-            ilog("Initialization error: requires at minimum 2 transfer accounts");
+         if(account_str_vector.size() < 1) {
+            ilog("Initialization error: requires at minimum 1 account");
```

Comment:
What necessitated this change?

Reply:
Troubleshooting, and I missed reverting this, fixed now. https://github.com/AntelopeIO/leap/pull/713/commits/724ba6cdae9cf9c3f7a7a1c8a92c88e468b4da3a